src/index.js

```diff
@@ -113,11 +113,20 @@ export { default as TableFooter } from './DataTable/TableFooter';
 export { default as CardView } from './DataTable/CardView';
 export { default as ToggleButton, ToggleButtonGroup } from './ToggleButton';
 export { default as Variant } from './utils/constants';
-export { default as useWindowSize } from './hooks/useWindowSize';
-export { default as useToggle } from './hooks/useToggle';
+
+// Custom hooks
+export {
+  useToggle,
+  useWindowSize,
+} from './hooks';
 
 // Pass through any needed whole third-party library functionality
 // useTable for example is needed to use the DataTable component seamlessly
 // rather than setting a peer dependency in this project, we opt to tightly
 // couple these dependencies by passing through needed functionality.
 export { useTable } from 'react-table';
+
+// ``ParagonProvider`` is a HOC that allows consumers of @edx/paragon to inject dependencies
```

Comment:
Nit: I don't think ParagonProvider is technically a higher order component, it just uses component composition.  The doc (https://reactjs.org/docs/higher-order-components.html) is somewhat vague on it, but I think the spirit of a HOC is that it wraps the component _definition_ and returns a wrapper that instantiates the component as part of whatever else it does.  i.e., `connect(MyComponent)` or `injectIntl(MyComponent)`.

That's a bit different than a component that takes `children` expressed as JSX and drops them into its render function.  I think that's just plain component composition, and the term "Provider" seems to have been adopted for components that do nothing else than provide some functionality to the ones composed in 'em.

Reply:
Yep, this isn't really an HOC; I will update the comment.